services/headless-lms/models/src/library/progressing.rs

```diff
@@ -48,6 +48,57 @@ pub async fn update_automatic_completion_status_and_grant_if_eligible(
             submodule_completions_required,
         )
         .await?;
+
```

Comment:
if completion_exists -> set boolean on the course_module_completions table "needs_to_be_reviewed"